src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -438,6 +432,15 @@ ucs_status_t uct_rc_verbs_ep_flush(uct_ep_h tl_ep, unsigned flags,
         }
     }
 
+    if (ucs_unlikely(flags & UCT_FLUSH_FLAG_CANCEL)) {
```

Comment:
```
if (CANCEL) {
 assert(pending is empty)
 if (available==max) {
     return ok
 }
 
 modify_to_err()
 add flush completion
}
```

Reply:
https://github.com/openucx/ucx/pull/5830#discussion_r513113682

Reply:
i withdraw that 👍
since modify to err will flush out unsignaled sends as well

Reply:
"performed before we moved QP to error"